src/lib/core/overlay/scroll/scroll-dispatcher.ts

```diff
@@ -105,7 +105,7 @@ export class ScrollDispatcher {
   getScrollContainers(elementRef: ElementRef): Scrollable[] {
     const scrollingContainers: Scrollable[] = [];
 
-    this.scrollableReferences.forEach((subscription: Subscription, scrollable: Scrollable) => {
+    this.scrollableReferences.forEach((_subscription: Subscription, scrollable: Scrollable) => {
```

Comment:
Unfortunately there is no `forEach()` method for iterators. And if we want to use iterators here we need to enable down-leveling of those ([read here](https://www.typescriptlang.org/docs/handbook/iterators-and-generators.html) and this requires a runtime shim for non-arrays.